cpu/rpx0xx/include/periph_cpu.h

```diff
@@ -432,6 +432,38 @@ typedef struct {
     uint8_t ch_numof;               /**< number of timer channels */
 } timer_conf_t;
 
+/**
+ * @name   I2C bitmasks
+ * @{
+*/
+#define IC_DATA_CMD_CMD 0x08
+#define IC_DATA_CMD_STOP 0x09
+/** @} */
+
+/**
+ * @name   Override I2C clock speed values
+ * @{
+ */
+#define HAVE_I2C_SPEED_T
+typedef enum {
+    I2C_SPEED_LOW       = 0x01,     /**< not supported               */
+    I2C_SPEED_NORMAL    = 100000U,  /**< normal mode:    ~100kbit/s  */
+    I2C_SPEED_FAST      = 400000U,  /**< fast mode:      ~400kbit/s  */
+    I2C_SPEED_FAST_PLUS = 1000000U, /**< fast mode plus: ~1000kbit/s */
+    I2C_SPEED_HIGH      = 0x03,     /**< not supported               */
+} i2c_speed_t;
+/** @} */
+
+/**
+ * @brief   I2C configuration options
+ */
+typedef struct {
+    uint32_t* dev; //device pointer should have better type probably
```

Comment:
Please use C Style /* */ only.
You can have a look at our coding convention [here](https://github.com/RIOT-OS/RIOT/blob/master/CODING_CONVENTIONS.md)

Reply:
Apologies, I will fix this.